Xamarin.PropertyEditing.Mac/Controls/RatioEditorControl.cs

```diff
@@ -26,10 +26,9 @@ public RatioEditorControl (IHostResourceProvider hostResources)
 			};
 			AddSubview (this.ratioEditor);
 
-			this.AddConstraints (new[] {
-				NSLayoutConstraint.Create (this.ratioEditor, NSLayoutAttribute.CenterY, NSLayoutRelation.Equal, this,  NSLayoutAttribute.CenterY, 1f, 0),
+			AddConstraints (new[] {
+				NSLayoutConstraint.Create (this.ratioEditor, NSLayoutAttribute.Bottom, NSLayoutRelation.Equal, this,  NSLayoutAttribute.Bottom, 1f, 0),
```

Comment:
No constant offset?

Reply:
Actually this highlights a bug, where the Base control's height was manually being set. This should be fixed now and BottomOffset added.